src/components/PromotedActionsBar.tsx

```diff
@@ -50,11 +50,16 @@ const PromotedActions = {
             ReportActions.joinRoom(report);
         }),
     }),
-    message: ({accountID, login}) => ({
+    message: ({reportID, accountID, login}) => ({
         key: CONST.PROMOTED_ACTIONS.MESSAGE,
         icon: Expensicons.CommentBubbles,
         text: Localize.translateLocal('common.message'),
         onSelected: () => {
+            if (reportID) {
+                Navigation.dismissModal(reportID);
```

Comment:
✋ Hey, coming from https://github.com/Expensify/App/issues/57673
This has caused an issue with navigation, this is a `MESSAGE` button, but when clicking it we were dismissing a modal instead of navigating to the correct report. This was resolved by changing `dismissModal` to `navigateToReportWithPolicyCheck`